Dockerfile

```
@@ -1,13 +1,13 @@
 FROM jinaai/jina:master as base
 
-COPY . ./image_normalizer/
-WORKDIR ./image_normalizer
+COPY . ./ranking_evaluator/
```

Does this work with `docker build`?
```suggestion
COPY . ./ranking_evaluator/
```

---

it works, what is the change requested?

---

my fault. It is ok